stock_reception_screen_measuring_device/models/measuring_device.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class MeasuringDevice(models.Model):
+    _inherit = "measuring.device"
+
+    is_default = fields.Boolean(
```

Comment:
maybe can be moved to the base module?

Reply:
we don't need the concept in the base module -> I'd rather keep it here, at least for now.